recipes/libxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class LibxlsxwriterConan(ConanFile):
+    name = "libxlsxwriter"
+    license = "FreeBSD"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jmcnamara/libxlsxwriter"
+    topics = ("conan", "Excel", "XLSX")
+    description = "A C library for creating Excel XLSX files"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "use_system_minizip": [True, False]
+    }
+    default_options = {
+        "shared": False, 
+        "fPIC": True,
+        "use_system_minizip": False,
+    }
+    exports_sources = ["CMakeLists.txt", "patches/*"]
+    generators = "cmake"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+        if self.options.use_system_minizip:
+            self.options["zlib"].minizip = True
```

Comment:
```suggestion
```

On Config options, the options are not solved, you must move to configure() instead.